src/applications/mhv/secure-messaging/components/MessageThread/MessageThreadItem.jsx

```diff
@@ -39,21 +40,21 @@ const MessageThreadItem = props => {
 
   const accordionAriaLabel = useMemo(
     () => {
-      return `${!isRead ? 'New ' : ''}message ${
+      return `${!isSentOrRead ? 'New ' : ''}message ${

```

Comment:
I find this kind of difficult to read, would it be worth doing some templating or otherwise making it more clear how logic maps into content? It might just be me 😊